examples/lammps/test.bats

```diff
@@ -85,5 +86,5 @@ lammps_try () {
 }
 
 @test "${ch_tag}/revert image" {
-    unpack_img_all_nodes "$ch_cray"
+    unpack_img_all_nodes "$ch_cray" "$ch_tag"
```

Comment:
Likewise, I think `unpack_img_all_nodes` can probably stick with the global `ch_tag`.